packages/blocks/src/text-input/index.js

```diff
@@ -10,11 +10,18 @@ import { isEmpty } from 'lodash';
  * Internal dependencies
  */
 import { useColorStyles } from '@crowdsignal/styles';
-import { ErrorMessage, FormInputWrapper, FormTextInput } from '../components';
+import {
+	ErrorMessage,
+	FormInputWrapper,
+	FormTextarea,
+	FormTextInput,
+} from '../components';
 import { useField } from '@crowdsignal/form';
 import validator from './validations';
 
 const TextInput = ( { attributes, className } ) => {
+	const MULTILINE_THRESHOLD = 70;
```

Comment:
Again, I'd place it outside of the function. Better yet, you could later assign it to `TextInput.MULTILINE_THRESHOLD` and use that inside the editor block rather than having two separate definitions for the same thing.